lib/puppet/provider/zone/solaris.rb

```diff
@@ -37,43 +53,42 @@ def self.instances
   # file or string.
   def configure
 
```

Comment:
We need to play well with others, unified archives aren't present until Solaris 11.2
$facts['os']['release'] major/minor or full should filter this type in defaultfor so it doesn't override the puppet provided zone provider for older releases.

os => {"name"=>"Solaris", "family"=>"Solaris", "release"=>{"major"=>"12", "minor"=>"0", "full"=>"12.0"}}